myeongseok/app.js

```diff
@@ -47,16 +52,17 @@ app.post('/users/signup', (req, res, next) => {
 });
 
 app.post('/posts', (req, res, next) => {
-  const { title, content, user_id } = req.body;
+  const { title, post_image, content, user_id } = req.body;
```

Comment:
```suggestion
  const { title, postImageUrl, content, user_id } = req.body;
```
- JavaScript에서는 기본적으로 `snake_case`가 아닌 `CamelCase`로 작성합니다.
- 또한 snake_case, CamelCase를 뒤죽박죽 섞어서 사용하면 안됩니다.
- 하나의 컨벤션으로 통일해서 작성해 주세요.

Reply:
세세하게 짚어주셔서 감사합니다!!